### ISSUE TITLE
3 Possible bugs regarding pronounciations and speech

### ISSUE THREAD

**Issue by TechHorseG** [nvaccess/nvda#10754]
Win10 / Version: 2019.2.1 / portable / MS George

1. I have noticed that NVDA does not read out smilies unless if they are reached by ctrl arrowing. For example, consider the text:

hello :) Having a good time?

That happy smilie is not read out whether I mouse over the text, or get there with the KB arrows.

If however I use ctrl and left / right to move between words on that line, then NVDA will read out "happy smilie" when the smilie is reached that way.

Is this a bug, or is there some setting I am missing? I want the smilie to always be read out irrespective of how NVDA gets to it.

2. How can you make NVDA read out a single "a" as "ay" rather than "ah"? I have tried several settings in the speech menu but can't seem to get this so perhaps I am missing something .

3. One final thing, if you consider the following sentence:

Ass is another word for donkey

I wonder if this is unique to those of us in the UK, but NVDA reads out that "ass" as "arse" as if it thinks it is refering to the part of the anatomy and is then applying the British pronounciationm. Is there anyway to stop this and make it jhust say "ass"? Again, have tried several settings such as dialect.

Thanks.

**Comment by Adriani90** [comment]
Hello,

bug 1: please try with nvda 2019.3 RC2 and see if the issue is reproducible. Have you tried with different synthesizers? This is something that is controlled by the synthesizers themselves in most cases.

Bug 2. You can create a new entry in the speech dictionary and define the exact pronounciation of a which you like.

Bug 3: this is definitely something controlled by the synthesizer itself, you should write to the synth developer. There is nothing NVDA can do here.


So please report if Bug 1 is still reproducible in NVDA 2019.3 RC2 with different synthesizers.

**Comment by Adriani90** [comment]
@TechHorseG after testing further, it seems your smily is composed of two characters. So NVDA does not render the two characters together because they could mean something else. For example someone would write something in brackets and typing : and then closing the bracket would make NVDA read smily which is not intended. There are already rendered characters in the CLDR repository for smilies and other emojies which are read properly by NVDA.
I am closing this issue because the pronounciation of the characters as emojy or something else is rather controlled by the synthesizer and not by NVDA.
